src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserResponseMapper userResponseMapper;
```

Comment:
Use abstraction (DtoRequestMapper and DtoResponseMapper) and in constructors as well

Reply:
I'll remove it because I don't use it, my register method is void